CONTRIBUTING.md

```diff
@@ -53,6 +53,90 @@ In general, we would like our deploy process to be: deploy new code + old config
 
 When you add a new RPC to a Boulder service (e.g. `SA.GetFoo()`), all components that call that RPC should wrap those calls in some flag. Generally this will be a boolean config field `UseFoo`. Since `UseFoo`'s zero value is false, a deploy with the existing config will not call `SA.GetFoo()`. Then, once the deploy is complete and we know that all SA instances support the `GetFoo()` RPC, we do a followup config deploy that sets `UseFoo()` to true.
 
+## Flag-gated migrations
+
+We use a [database migrations](https://en.wikipedia.org/wiki/Schema_migration)
+to modify the existing schema. These migrations will be run on live
+data while Boulder is still running, we need Boulder code at any given commit to
```

Comment:
I feel like the first part is the explanation of how we apply the migrations to a live system and the latter part is the implication that we need Boulder to handle the change gracefully. Maybe a "therefore" after the comma to better express the cause/effect?